src/sdk/pynni/nni/msg_dispatcher.py

```diff
@@ -108,6 +108,9 @@ def handle_request_trial_jobs(self, data):
     def handle_update_search_space(self, data):
         self.tuner.update_search_space(data)
 
+    def handle_feed_tuner_data(self, data):
```

Comment:
tuner or tuning? This function is also used by the Advisor.

Reply:
In this specific dispatcher, data is fed to tuner. Advisor is another dispatcher, where he can do whatever he wants.

Reply:
You mean you want to have two functions? one "handle_feed_tuner_data", one "handle_feed_advisor_data"? 
But they are actually doing a similar thing?
So why not all use "handle_feed_tuning_data"?

Reply:
Have renamed it